### DISCUSSION TITLE
Q-tree receiving data

### DISCUSSION THREAD

**Discussion by Felipe-Tomazetti** [quasarframework/quasar#13350] [General - Components / Directives / etc] [answered]
I'm a bit new to the developer life, and I'm using Q-tree strict mode from Quasar, I have no idea how do I pass the ticked nodes to the ticket array, below follows some screenshots of the code

![1](https://user-images.githubusercontent.com/48369245/166813105-6eeb65b9-9bc2-4f0b-80f2-d377f11f9c8b.png)
![2](https://user-images.githubusercontent.com/48369245/166813113-d55e0ee9-48dd-436c-9942-d848f3ffda68.png)
![3](https://user-images.githubusercontent.com/48369245/166813206-f43296c4-dc30-412e-ae88-82f03cf50b4d.png)

**Comment by metalsadman** [comment, marked as answer]
take this as an example https://quasar.dev/vue-components/tree#example--tick-strategy

**Reply by Felipe-Tomazetti** [reply]
Is there a way that I can pass the ticked values to a function? Like the _id of the ticket value

**Reply by metalsadman** [reply]
you can with `@update:ticked="someHandler"`, also use a template ref on your QTree so you can access its methods in this case `getNodeByKey`.
```js
function someHandler(key) {
  const fullNodeObj = myQtreeRef.value.getNodeByKey(key)
  console.log(fullNodeObj._id)
}
```
check the api card https://quasar.dev/vue-components/tree#qtree-api
